src/include/sof/audio/component.h

```diff
@@ -64,21 +64,24 @@ struct timestamp_data;
 #define COMP_STATE_PREPARE	3	/**< Component prepared */
 #define COMP_STATE_PAUSED	4	/**< Component paused */
 #define COMP_STATE_ACTIVE	5	/**< Component active */
+#define COMP_STATE_STANDBY	6	/**< Component after early initialisation */
```

Comment:
STANDBY probably not  best naming PRE_ACTIVE is better (also means we can have POST_ACTIVE if needed).

Reply:
`STANDBY` is definitely not the best :-)